nifi-mock/src/test/java/org/apache/nifi/util/TestMockProcessSession.java

```diff
@@ -36,7 +36,7 @@ public void testPenalizeFlowFileFromProcessor() {
 
     protected static class PoorlyBehavedProcessor extends AbstractProcessor {
 
-        private static final Relationship REL_FAILURE = new Relationship.Builder()
+        protected static final Relationship REL_FAILURE = new Relationship.Builder()
```

Comment:
Directly using it in Test NonExistentRelationTestProcessor public Set<Relationship> relationships = Collections.singleton(REL_FAILURE);